bndtools.m2e/src/bndtools/m2e/MavenRunListenerHelper.java

```diff
@@ -34,11 +33,12 @@
 
 public interface MavenRunListenerHelper {
```

Comment:
Does this type still need to be an interface since all the methods are now static? Perhaps it should be a final class with a private constructor?

Reply:
It was previously an interface used for a sort of horrible multiple inheritance. It’s another type that would look very different after a proper refactor. 
I’m not claiming that this commit is beautiful, it’s a sticking plaster.